src/main/java/io/strimzi/kafka/bridge/KafkaBridgeAdmin.java

```diff
@@ -84,6 +86,29 @@ public CompletionStage<Set<String>> listTopics() {
         return promise;
     }
 
+    /**
+     * Creates a topic with given name
+     *
+     * @param topicName topic name to create
+     * @return a CompletionStage Void
+     */
+    public CompletionStage<Void> createTopic(String topicName) {
+        LOGGER.trace("Create topic thread {}", Thread.currentThread());
+        LOGGER.info("Create topic {}", topicName);
+        CompletableFuture<Void> promise = new CompletableFuture<>();
+        this.adminClient.createTopics(Collections.singletonList(new NewTopic(topicName, 2, (short) 1)))
```

Comment:
why are partitions and replication factor hard coded? They should be parameters passed through the API imho.

Reply:
Sorry I missed this, will be fixed 👍